hack/istio/enable_maistra_deployment_injection.sh

```diff
@@ -0,0 +1,53 @@
+#/bin/bash
+
+# This enables deployment sidecar injection for Maistra.  The deployments should already exist in the target namespace.
+
+# Given a namepace, prepare the deployments for injection
+# This means:
+# 1. Annotate all of the namespace's Deployments with the sidecar injection annotation
+prepare_maistra() {
+  local ns="${1}"
+
+  for d in $(oc get deployments -n ${ns} -o name)
+  do
+    echo "Enabling sidecar injection for deployment: ${d}"
+    oc patch ${d} -n ${ns} -p '{"spec":{"template":{"metadata":{"annotations":{"sidecar.istio.io/inject": "true"}}}}}' --type=merge
```

Comment:
I recently asked the Istio team about this and they told me the label should be used now. This is in the docs, also (using label, not annotation) - see: https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#controlling-the-injection-policy

```suggestion
    oc patch ${d} -n ${ns} -p '{"spec":{"template":{"metadata":{"labels":{"sidecar.istio.io/inject": "true"}}}}}' --type=merge
```